src/controllers/UserController.ts

```diff
@@ -19,4 +20,20 @@ export default class UserController {
   async createUser(@Body() user: User): Promise<User> {
     return this.userService.createUser(user);
   }
+
+  @Put(':userId')
+  @Logged()
+  updateUserById(@Param('userId') userId: string, @Body() user: User): Promise<User> {
+    const updatedUser = {
+      ...user,
+      id: userId,
+    };
+    return this.userService.updateUserById(userId, updatedUser);
```

Comment:
Is it necessary to pass two parameters?

Reply:
It clearly defines the method functionality & differentiates the update object with userId.